### DISCUSSION TITLE
WCAG 2.2 AAA* audit of the Wagtail admin – Nov 2023

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/wagtail#11180] [Development]
👋 opening this for discussion of an upcoming WCAG audit of the admin. We’re looking for feedback / guidance / suggestions (or even just kind words of support!) on how we plan the audit, specifics of the testing, and how we share reports / tackle the issues.

## Context

As part of our [ATAG 2.0 audit](https://github.com/wagtail/wagtail/issues/10515), we succinctly reviewed the accessibility of the Wagtail admin. There were clear gaps so for WCAG conformance we focused on automated tests, and spent our manual testing budget on other ATAG criteria.

Now – we need to go back to WCAG auditing, and test the relevant parts of the admin more meaningfully.

### Standards

For _Guideline A.1.1:_ of ATAG 2.0 – the audit should include all of [WCAG _2.2_ AA-level](https://www.w3.org/TR/WCAG22/) (Recommendation as of October 2023), and some of AAA-level guidelines as well as best practice recommendations as we see fit. We could also attempt to cover a few concepts from WCAG 3.0 (draft), such as color contrast. So the options are:

- WCAG 2.2 AA
- WCAG 2.2 AAA (TBC which SCs)
- WCAG 3.0 (draft)

We could also possibly try to cover legal requirements from:

- Section 508 (USA)
- EN 301 549 (EU)
- PSBAR 2018 (UK)

### Scope

For ease of auditing and of reviewing / sharing audit results, I would like us to focus on features of the admin accessible via one of my static "snapshot" copies of Wagtail: <https://static-wagtail-v5-2.netlify.app/admin/>. This limits our possible scope but greatly simplifies having access to features and sharing links to specific parts of Wagtail when reporting issues.

From there have a lot of options for how we go about this. Here are the ones I could think of:

- Wagtail admin views as defined via [UI mapping spreadsheet](https://docs.google.com/spreadsheets/d/1WaqARpHf99U0O94hypwHNjHA9yNpn4AnXnIT5AdJsm8/edit)
- Wagtail components as defined via [UI mapping spreadsheet](https://docs.google.com/spreadsheets/d/1WaqARpHf99U0O94hypwHNjHA9yNpn4AnXnIT5AdJsm8/edit)
- Wagtail CMS features [in use in the bakerydemo website](https://gist.github.com/thibaudcolas/f1507030e5e263592ff3d10c80b00ffe)
- Representative sample selected based on [WCAG-EM](https://www.w3.org/WAI/test-evaluate/conformance/wcag-em/)
- CMS features flagged as [accessibility features needing documentation](https://wagtail.org/accessibility/atag-audit/#a421-describe-accessibility-features)

### Reporting

We have a lot of options here:

- [Living accessibility audit spreadsheet](https://docs.google.com/spreadsheets/d/1l7tnpEyJiC5BWE_JX0XCkknyrjxYA5T2aee5JgPnmi4/edit), or similar spreadsheet
- Direct to GitHub issues
- [Be Inclusive](https://beinclusive.app/)
- <https://github.com/hidde/eleventy-wcag-reporter>
- OpenACR / VPAT

### Testing methodology

Essentially TBC, but I would like us to focus on manual testing, since we got lots of reports from automated tests already.

#### Semi-automated WCAG testing

Similar to past accessibility team reviews of Wagtail 3.0 and 4.0 via screenshots in Figma.

- Screenshot-taking & visual review in light mode for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))
- Screenshot-taking & visual review in dark mode for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))
- Screenshot-taking & visual review in forced-colors mode (WHCM) - light for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))
- Screenshot-taking & visual review in forced-colors mode (WHCM) - dark for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))
- Screenshot-taking & visual review for landmarks and headings for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))
- Screenshot-taking & visual review for accessible labels for 100% of scope views in their default state ([wagtail-tooling](https://github.com/thibaudcolas/wagtail-tooling))

#### Manual WCAG testing

Across a representative sample of admin views,

- Screenshot-taking & visual review for tab order
- Manual keyboard navigation testing
- Manual touch navigation testing
- Manual 400% zoom testing
- Desktop screen readers: VoiceOver macOS, NVDA, JAWS
- Mobile screen readers: VoiceOver iOS, TalkBack
- Speech recognition: Dragon, Voice Control
- Magnification with Apple Zoom
- WHCM
- Print stylesheets

## Audit planning

- Validate requirements & decide on approach
- Confirm availability of relevant stakeholders / auditors
- Audit
- Report
- Backlog curation
- Blog post sharing / social media

**Comment by thibaudcolas** [comment]
## Meeting notes – 2023-11-03 10:15

### Actions

#### Albina

- ✅ @albinazs decide on which views of the CMS to audit (2-3h top 10)
- ✅ @albinazs review possible standards to target, decide on AA vs. AAA, possible WCAG 3.0 inclusion – 0.5h 
- ✅ @albinazs review example of [OpenACR](https://gsa.github.io/openacr/) (1h)
- ✅ @albinazs decide whether we audit the [German](https://static-wagtail-v5-2.netlify.app/admin-german/pages/62/edit/) / English / Arabic versions of the demo – English
- ✅ @albinazs review [list of accessibility features to document](https://github.com/thibaudcolas/rfcs/blob/99-atag-audit/text/089-atag-audit.md#a421-describe-accessibility-features) to pick the ones we want to audit – 0.5h / 1h
- ✅ @albinazs Research what the gold standard is for browser zoom testing (640px 200%, 1280px 400%) and share your findings in #accessibility (1h)

#### Thibaud

- @thibaudcolas Decide which automated tools we use for auditing (0.5h)
- @thibaudcolas Update spreadsheet template for WCAG 2.2 (0.25h)
- @thibaudcolas Fix demo site to work in light mode (0.5h)

##### Later

- @thibaudcolas Fix german demo site for v5.2 (0.25h)
- @thibaudcolas update links to UI mapping sheets: https://docs.google.com/spreadsheets/d/1FMSA_BI3ZvkeAvuaIL2QtqRTgMNwz_vhfKBeyx2Onnk/edit  (0.25h)
- @thibaudcolas Review Living™ accessibility audit and make sure it’s usable (0.25h)
- @thibaudcolas Review executive summary template to make sure it’s up to date and suitable for Wagtail (0.25h)
- @thibaudcolas Get a JAWS license
- @thibaudcolas Ask accessibility team input (2h)
- @thibaudcolas Accessibility team meeting to review plan ahead of start
- @thibaudcolas Update screenshot taking tooling (3h)
- @thibaudcolas Do another round of automated screenshot taking for WHCM (3h)
- @thibaudcolas blog post screenshot-automation methodology (3h)

### Decisions

#### Scope

- English version of the admin
- German / Arabic if there is time

##### Views

Views to test:

- Dashboard and global components / navigation
  - Error, Success, Warning messages as seen in Styleguide
- Page editor _for snippets_: https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/edit/1/
- Page editor _for pages_: Hot Cross Brun except for the Body field due to complexity (https://static-wagtail-v5-2.netlify.app/admin/pages/81/edit/)

#### Reporting

- Living accessibility audit for Albina to track issues
- (Then GitHub issues)
- (Executive summary: Torchbox executive summary template)

#### Testing methodology

Must:

- Accessibility insights
- WHCM / contrast themes
- Manual keyboard navigation testing
- VoiceOver iOS + Manual touch navigation testing

Should:

- VoiceOver + Safari
- Print stylesheets
- Voice Control

Could:

- NVDA + Firefox
- Manual 400% zoom testing (1280px 400% -> 320px)
- JAWS (license-dependent)

**Comment by ArafatAbdussalam** [comment]
Hi @thibaudcolas I will like to participate in the Accessibility audit testing. This will enable me to be able to gain further experience on Accessibility. Having read about the implementation of ATAG, it will be more exciting to be able to demonstrate some skills for a successful audit.

**Reply by thibaudcolas** [reply]
hi @ArafatAbdussalam 👋 Based on [my comment below](https://github.com/wagtail/wagtail/discussions/11180#discussioncomment-7906101), are you still keen to help with this?

If so perhaps you could test the following views for us?

| Functionality          | URL                                                                                                                         |
| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- |
| Page not found (404)   | [/admin/404/](https://static-wagtail-v5-2.netlify.app/admin/404/)                                                           |
| **Forms**              |                                                                                                                             |
| View all               | [/admin/forms/](https://static-wagtail-v5-2.netlify.app/admin/forms/)                                                       |
| View submissions       | [/admin/forms/submissions/69/](https://static-wagtail-v5-2.netlify.app/admin/forms/submissions/69/)                         |

**Reply by ArafatAbdussalam** [reply]
Hi @thibaudcolas Yes, sure. I am still keenly interested in the audit. I have been taking my time to understand the next step to be taken but I am quite lost as I was unable to attend the last meeting.

**Comment by yigakpoa** [comment]
Hi @thibaudcolas I will love to work with the team on this!

**Reply by thibaudcolas** [reply]
hi @yigakpoa  👋 Based on [my comment below](https://github.com/wagtail/wagtail/discussions/11180#discussioncomment-7906101), do you still want to do this? Here are views I think you could help us test:

| Functionality          | URL                                                                                                                         |
| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- |
| **Auth**               |                                                                                                                             |        |
| Login                  | [/admin/login/](https://static-wagtail-v5-2.netlify.app/admin/login/)                                                       |        |
| Password reset         | [/admin/password_reset/](https://static-wagtail-v5-2.netlify.app/admin/password_reset/)                                     |        |
| Password reset done    | [/admin/password_reset/done/](https://static-wagtail-v5-2.netlify.app/admin/password_reset/done/)                           |        |

**Comment by SaptakS** [comment]
@thibaudcolas let me know where I can help with the auditing.

**Reply by thibaudcolas** [reply]
@SaptakS 🙌 please see [my comment below](https://github.com/wagtail/wagtail/discussions/11180#discussioncomment-7906101).

If you want to help with this, I think we could use extra expertise on testing the user bar: #10674

**Reply by SaptakS** [reply]
I will take a look at it

**Comment by praiztech** [comment]
@thibaudcolas I'd love to be on the auditing team.

**Reply by thibaudcolas** [reply]
hi @praiztech   👋 Based on [my comment below](https://github.com/wagtail/wagtail/discussions/11180#discussioncomment-7906101), do you still want to do this? Here are views I think you could help us test:

| Functionality          | URL                                                                                                                         |
| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- |
| **Promoted search**    |                                                                                                                             |        |
| View all               | [/admin/searchpicks/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/)                                           |        |
| Edit                   | [/admin/searchpicks/5/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/5/)                                       |        |
| Add results            | [/admin/searchpicks/add/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/add/)                                   |        |
| Delete                 | [/admin/searchpicks/5/delete/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/5/delete/)                         |        |

Alternatively, since you have much more experience with this than most, I’d be keen to possibly give you an assignment that requires more autonomy? Let us know what your plans are.

**Reply by praiztech** [reply]
@thibaudcolas Sure! Let me know what the assignment is about.

**Reply by thibaudcolas** [reply]
@praiztech could you read my comment below that I linked to and let me know if you have questions? This is an accessibility audit of Wagtail, we’ve detailed the methodology towards the top of this discussion thread and with the recent update from @albinazs. For you specifically, I shared four different views in the CMS right above that you could help test.

**Reply by praiztech** [reply]
@thibaudcolas my comment was in response to your asking if I was available for an assignment.

**Comment by ShubhamOulkar** [comment]
Hi @thibaudcolas I will like to participate in the Accessibility audit testing.

**Reply by thibaudcolas** [reply]
hi @ShubhamOulkar  👋 Based on [my comment below](https://github.com/wagtail/wagtail/discussions/11180#discussioncomment-7906101), do you still want to do this? Here are views I think you could help us test:

| Functionality          | URL                                                                                                                         |
| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- |
| **Reports**            |                                                                                                                             |        |
| Locked Pages           | [/admin/reports/locked/](https://static-wagtail-v5-2.netlify.app/admin/reports/locked/)                                     |        |
| Workflows              | [/admin/reports/workflow/](https://static-wagtail-v5-2.netlify.app/admin/reports/workflow/)                                 |        |
| Workflow tasks         | [/admin/reports/workflow_tasks/](https://static-wagtail-v5-2.netlify.app/admin/reports/workflow_tasks/)                     |        |
| Site history           | [/admin/reports/site-history/](https://static-wagtail-v5-2.netlify.app/admin/reports/site-history/)                         |        |
| Aging pages            | [/admin/reports/aging-pages/](https://static-wagtail-v5-2.netlify.app/admin/reports/aging-pages/)                           |        |

**Comment by albinazs** [comment]
## Update 13-12-2023

[Spreadsheet with audit output](https://docs.google.com/spreadsheets/d/1A31FI2hFRzG6oocZabCpeAIvty5f5fk2uqdbr3R9yLA/edit?usp=sharing) (still WIP)

#### Pages checked:
- [Dashboard](https://static-wagtail-v5-2.netlify.app/admin/)
- [Styleguide](https://static-wagtail-v5-2.netlify.app/admin/styleguide/)
- [Page editor (snippets)](https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/edit/1/)
- [Page editor (page, all except body)](https://static-wagtail-v5-2.netlify.app/admin/pages/81/edit/)

#### Testing methods:
- Semi-automated checks with [Accessibility Insights](https://accessibilityinsights.io/) and [Sa11y](https://sa11y.netlify.app/)
- Manual keyboard navigation
- [Contrast themes](https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696) (emulated in Chrome dev tools)

This audit hasn’t produced as many significant issues as the previous one due to ongoing accessibility improvements. There are still some things to consider, mostly regarding the new rules of best practices that go beyond WCAG.

**Reply by ArafatAbdussalam** [reply]
Hi @albinazs What tool do we get to use for the audit testing? Following through with your comments. Can I use Accessibility insights and Sally as well?

**Reply by thibaudcolas** [reply]
@ArafatAbdussalam yes, please use the same tools as Albina did

**Comment by thibaudcolas** [comment]
Hi all, I have updated our [living accessibility audit](https://docs.google.com/spreadsheets/d/1l7tnpEyJiC5BWE_JX0XCkknyrjxYA5T2aee5JgPnmi4/edit) with the results from Albina’s work. As we can see in the "Scope overview" tab, there are a lot of pages within the CMS that remain untested simply due to time constraints. 

## Contributing to the audit

For anyone wanting to help with this, please:

1. Pick a page or pages you would want to test
2. Go through as many of our testing methods discussed here as seems reasonable to you based on your expertise
3. Report the results in this discussion

As far as reporting the results of your testing, please make a copy of our spreadsheet like Albina did, then adding your findings there, and share the link to the sheet in this thread.

## Where to test

We use the <https://static-wagtail-v5-2.netlify.app/admin/> website to do our testing for this audit. It’s a _static copy_ of Wagtail, not a real site instance. This helps a lot in making sure we all test the same thing, but does mean not all functionality in the CMS is testable this way.

## What to test

Here is a list of available views in our demo site, if you’re keen to help let us know here and I’ll keep track of who tests what in the one table :)

| Functionality          | URL                                                                                                                         | Tester |
| ---------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------ |
| **Global**             |                                                                                                                             |        |
| Dashboard              | [/admin/](https://static-wagtail-v5-2.netlify.app/admin/)                                                                   | @albinazs       |
| Page not found (404)   | [/admin/404/](https://static-wagtail-v5-2.netlify.app/admin/404/)                                                           |        |
| Wagtail userbar        | [Public-facing site](https://static-wagtail-v5-2.netlify.app/)                                                              |  @SaptakS       |
| **Pages**              |                                                                                                                             |        |
| Pages explorer         | [/admin/pages/60/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/)                                                 |        |
| Add child page         | [/admin/pages/60/add_subpage/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/add_subpage/)                         |        |
| Page type usage        |                                                                                                                             |        |
| Privacy                |                                                                                                                             |        |
| Move pages             | [/admin/pages/69/move/](https://static-wagtail-v5-2.netlify.app/admin/pages/69/move/)                                       |        |
| Copy                   | [/admin/pages/60/copy/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/copy/)                                       |        |
| Delete                 | [/admin/pages/60/delete/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/delete/)                                   |        |
| Publish                |                                                                                                                             |        |
| Unpublish              | [/admin/pages/60/unpublish/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/unpublish/)                             |        |
| View all revisions     | [/admin/pages/60/history/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/history/)                                 |        |
| Compare revisions      | [/admin/pages/60/history/compare/2...107/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/history/compare/2...107/) |        |
| Review revision        | [/admin/pages/60/history/2/revert/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/history/2/revert/)               |        |
| Create                 | [/admin/pages/add/base/homepage/60/](https://static-wagtail-v5-2.netlify.app/admin/pages/add/base/homepage/60/)             |        |
| Edit                   | [/admin/pages/60/edit/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/edit/)                                       | @albinazs       |
| **Images**             |                                                                                                                             |        |
| View all               | [/admin/images/](https://static-wagtail-v5-2.netlify.app/admin/images/)                                                     |        |
| Edit                   | [/admin/images/47/](https://static-wagtail-v5-2.netlify.app/admin/images/47/)                                               |        |
| Add an image           | [/admin/images/add/](https://static-wagtail-v5-2.netlify.app/admin/images/add/)                                             |        |
| Add multiple images    |                                                                                                                             |        |
| Delete                 | [/admin/images/47/delete/](https://static-wagtail-v5-2.netlify.app/admin/images/47/delete/)                                 |        |
| Image URL generator    | [/admin/images/47/generate_url/](https://static-wagtail-v5-2.netlify.app/admin/images/47/generate_url/)                     |        |
| **Documents**          |                                                                                                                             |        |
| View all               | [/admin/documents/](https://static-wagtail-v5-2.netlify.app/admin/documents/)                                               |        |
| Edit                   | [/admin/documents/edit/1/](https://static-wagtail-v5-2.netlify.app/admin/documents/edit/1/)                                 |        |
| Add a document         |                                                                                                                             |        |
| Add multiple documents | [/admin/documents/delete/1/](https://static-wagtail-v5-2.netlify.app/admin/documents/delete/1/)                             |        |
| Delete                 | [/admin/documents/multiple/add/](https://static-wagtail-v5-2.netlify.app/admin/documents/multiple/add/)                     |        |
| **Snippets**           |                                                                                                                             |        |
| View all types         | [/admin/snippets/](https://static-wagtail-v5-2.netlify.app/admin/snippets/)                                                 |        |
| View all               | [/admin/snippets/base/person/](https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/)                         |        |
| Edit                   | [/admin/snippets/base/person/add/](https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/add/)                 | @albinazs       |
| Add                    | [/admin/snippets/base/person/delete/1/](https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/delete/1/)       |        |
| Delete                 | [/admin/snippets/base/person/edit/1/](https://static-wagtail-v5-2.netlify.app/admin/snippets/base/person/edit/1/)           |        |
| **Forms**              |                                                                                                                             |        |
| View all               | [/admin/forms/](https://static-wagtail-v5-2.netlify.app/admin/forms/)                                                       |        |
| View submissions       | [/admin/forms/submissions/69/](https://static-wagtail-v5-2.netlify.app/admin/forms/submissions/69/)                         |        |
| **Reports**            |                                                                                                                             |        |
| Locked Pages           | [/admin/reports/locked/](https://static-wagtail-v5-2.netlify.app/admin/reports/locked/)                                     |        |
| Workflows              | [/admin/reports/workflow/](https://static-wagtail-v5-2.netlify.app/admin/reports/workflow/)                                 |        |
| Workflow tasks         | [/admin/reports/workflow_tasks/](https://static-wagtail-v5-2.netlify.app/admin/reports/workflow_tasks/)                     |        |
| Site history           | [/admin/reports/site-history/](https://static-wagtail-v5-2.netlify.app/admin/reports/site-history/)                         |        |
| Aging pages            | [/admin/reports/aging-pages/](https://static-wagtail-v5-2.netlify.app/admin/reports/aging-pages/)                           |        |
| **Workflows**          |                                                                                                                             |        |
| View all               | [/admin/workflows/list/](https://static-wagtail-v5-2.netlify.app/admin/workflows/list/)                                     |        |
| Add                    |                                                                                                                             |        |
| Usage                  |                                                                                                                             |        |
| Edit                   | [/admin/workflows/edit/1/](https://static-wagtail-v5-2.netlify.app/admin/workflows/edit/1/)                                 |        |
| **Workflow tasks**     |                                                                                                                             |        |
| View all               | [/admin/workflows/tasks/index/](https://static-wagtail-v5-2.netlify.app/admin/workflows/tasks/index/)                       |        |
| Add                    |                                                                                                                             |        |
| Edit                   | [/admin/workflows/tasks/edit/1/](https://static-wagtail-v5-2.netlify.app/admin/workflows/tasks/edit/1/)                     |        |
| Disable                | [/admin/workflows/tasks/disable/1/](https://static-wagtail-v5-2.netlify.app/admin/workflows/tasks/disable/1/)               |        |
| **Users**              |                                                                                                                             |        |
| View all               | [/admin/users/](https://static-wagtail-v5-2.netlify.app/admin/users/)                                                       |        |
| Edit                   | [/admin/users/3/](https://static-wagtail-v5-2.netlify.app/admin/users/3/)                                                   |        |
| Edit - Edit roles      |                                                                                                                             |        |
| Add                    | [/admin/users/add/](https://static-wagtail-v5-2.netlify.app/admin/users/add/)                                               |        |
| Add - Add roles        |                                                                                                                             |        |
| Delete                 | [/admin/users/4/delete/](https://static-wagtail-v5-2.netlify.app/admin/users/4/delete/)                                     |        |
| **Groups**             |                                                                                                                             |        |
| View all               | [/admin/groups/](https://static-wagtail-v5-2.netlify.app/admin/groups/)                                                     |        |
| Add a group            | [/admin/groups/new/](https://static-wagtail-v5-2.netlify.app/admin/groups/new/)                                             |        |
| Edit                   | [/admin/groups/1/](https://static-wagtail-v5-2.netlify.app/admin/groups/1/)                                                 |        |
| Delete                 | [/admin/groups/1/delete/](https://static-wagtail-v5-2.netlify.app/admin/groups/1/delete/)                                   |        |
| Group users            |                                                                                                                             |        |
| **Sites**              |                                                                                                                             |        |
| View all               | [/admin/sites/](https://static-wagtail-v5-2.netlify.app/admin/sites/)                                                       |        |
| Edit                   | [/admin/sites/2/](https://static-wagtail-v5-2.netlify.app/admin/sites/2/)                                                   |        |
| Add a site             | [/admin/sites/new/](https://static-wagtail-v5-2.netlify.app/admin/sites/new/)                                               |        |
| Delete                 | [/admin/sites/2/delete/](https://static-wagtail-v5-2.netlify.app/admin/sites/2/delete/)                                     |        |
| **Locales**            |                                                                                                                             |        |
| View all               | [/admin/locales/](https://static-wagtail-v5-2.netlify.app/admin/locales/)                                                   |        |
| Edit                   |                                                                                                                             |        |
| Add a locale           |                                                                                                                             |        |
| Delete                 |                                                                                                                             |        |
| **Collections**        |                                                                                                                             |        |
| View all               | [/admin/collections/](https://static-wagtail-v5-2.netlify.app/admin/collections/)                                           |        |
| Edit                   | [/admin/collections/2/](https://static-wagtail-v5-2.netlify.app/admin/collections/2/)                                       |        |
| Add a collection       | [/admin/collections/add/](https://static-wagtail-v5-2.netlify.app/admin/collections/add/)                                   |        |
| Privacy                |                                                                                                                             |        |
| **Redirects**          |                                                                                                                             |        |
| View all               | [/admin/redirects/](https://static-wagtail-v5-2.netlify.app/admin/redirects/)                                               |        |
| Import redirects       |                                                                                                                             |        |
| Add                    | [/admin/redirects/add/](https://static-wagtail-v5-2.netlify.app/admin/redirects/add/)                                       |        |
| Export Redirects       |                                                                                                                             |        |
| Edit                   | [/admin/redirects/1/](https://static-wagtail-v5-2.netlify.app/admin/redirects/1/)                                           |        |
| Delete                 |                                                                                                                             |        |
| **Promoted search**    |                                                                                                                             |        |
| View all               | [/admin/searchpicks/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/)                                           |        |
| Edit                   | [/admin/searchpicks/5/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/5/)                                       |        |
| Add results            | [/admin/searchpicks/add/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/add/)                                   |        |
| Delete                 | [/admin/searchpicks/5/delete/](https://static-wagtail-v5-2.netlify.app/admin/searchpicks/5/delete/)                         |        |
| **User account**       |                                                                                                                             |        |
| Account profile        | [/admin/account/](https://static-wagtail-v5-2.netlify.app/admin/account/)                                                   |        |
| Account notifications  |                                                                                                                             |        |
| **Styleguide**         |                                                                                                                             |        |
| Styleguide             | [/admin/styleguide/](https://static-wagtail-v5-2.netlify.app/admin/styleguide/)                                             | @albinazs       |
| **Auth**               |                                                                                                                             |        |
| Login                  | [/admin/login/](https://static-wagtail-v5-2.netlify.app/admin/login/)                                                       |        |
| Password reset         | [/admin/password_reset/](https://static-wagtail-v5-2.netlify.app/admin/password_reset/)                                     |        |
| Password reset done    | [/admin/password_reset/done/](https://static-wagtail-v5-2.netlify.app/admin/password_reset/done/)                           |        |

**Reply by nandini584** [reply]
hey I would like to test [/admin/pages/60/](https://static-wagtail-v5-2.netlify.app/admin/pages/60/) this page, if its available? Also could you let me know some examples where I can have a look for the same?

**Reply by ArafatAbdussalam** [reply]
Alright @thibaudcolas Thank you for the resources.
